src/core/Menu/Menu.tsx

```diff
@@ -17,10 +18,24 @@ export type MenuProps = {
   role?: string;
   /**
    * Menu items. Recommended to use `MenuItem` components.
+   *
+   * If you have custom actionable items, they should have `tabindex={-1}` for better keyboard navigation support
+   * and selected item should have `aria-selected={true}`.
    */
   children: React.ReactNode;
 } & Omit<CommonProps, 'title'>;
 
+const getFocusableItems = (container: HTMLElement | null) => {
```

Comment:
Now I'm thinking this should be a common util function so we can use it in more components. We can rename the existing one to `getTabbableElements`.
![image](https://user-images.githubusercontent.com/9084735/134571809-3dee14ee-ceea-4787-be1e-b57b503c4a03.png)

Reply:
Added.